README.md

```diff
@@ -224,13 +225,13 @@ effects, you only have to specify the thresholds that you want to apply.
 <details><summary>Config Examples</summary>
 
 ```yaml
-# Take all uniques with item power > 800 
+# Take all uniques with item power > 800
```

Comment:
I'm assuming this is from the settings being applied from the readme
"Setup VS Code by using the black formater extension. Also turn on "trim trailing whitespaces" is VS Code settings."